### DISCUSSION TITLE
Error when 'make' command is given

### DISCUSSION THREAD

**Discussion by Shrihari01** [The-OpenROAD-Project/OpenLane#1673] [Q&A] [answered]
When 'make' command is given , it's showing 
FileNotFoundError:[Error 2] No such file or directory:'docker' : 'docker'
[Makefile:60:pull-run-base] Error 1
[Makefile:107::openlane] Error 2
[Makefile:117: get-openlane] Error 2

**Comment by vijayank88** [comment, marked as answer]
If OpenLane successfully installed from here mean: https://github.com/The-OpenROAD-Project/OpenLane

you should use `make mount` to enter into docker container. 

To simply test your installation `make test` directly.

Read this document: https://openlane.readthedocs.io/en/latest/
